sympy/tensor/tests/test_indexed.py

```diff
@@ -32,104 +54,105 @@ def test_Idx_bounds():
     i, a, b = symbols('i a b', integer=True)
     assert Idx(i).lower is None
     assert Idx(i).upper is None
-    assert Idx(i, a).lower == 0
-    assert Idx(i, a).upper == a - 1
-    assert Idx(i, 5).lower == 0
-    assert Idx(i, 5).upper == 4
-    assert Idx(i, oo).lower == 0
-    assert Idx(i, oo).upper == oo
-    assert Idx(i, (a, b)).lower == a
-    assert Idx(i, (a, b)).upper == b
-    assert Idx(i, (1, 5)).lower == 1
-    assert Idx(i, (1, 5)).upper == 5
-    assert Idx(i, (-oo, oo)).lower == -oo
-    assert Idx(i, (-oo, oo)).upper == oo
+    assert_equal(Idx(i, a).lower, 0)
+    assert_equal(Idx(i, a).upper, a - 1)
+    assert_equal(Idx(i, 5).lower, 0)
+    assert_equal(Idx(i, 5).upper, 4)
+    assert_equal(Idx(i, oo).lower, 0)
+    assert_equal(Idx(i, oo).upper, oo)
+    assert_equal(Idx(i, (a, b)).lower, a)
+    assert_equal(Idx(i, (a, b)).upper, b)
+    assert_equal(Idx(i, (1, 5)).lower, 1)
+    assert_equal(Idx(i, (1, 5)).upper, 5)
+    assert_equal(Idx(i, (-oo, oo)).lower, -oo)
+    assert_equal(Idx(i, (-oo, oo)).upper, oo)
```

Comment:
Can you move style changes like this one into a separate PR, please?
Such separation keeps workload off the shoulders of the math experts.